src/i_sdlmusic.c

```diff
@@ -649,8 +649,14 @@ static boolean ReadSubstituteConfig(char *filename)
 
     while (!feof(fs))
     {
+        char *retval;
         M_StringCopy(line, "", sizeof(line));
-        fgets(line, sizeof(line), fs);
+        retval = fgets(line, sizeof(line), fs);
+
+        if (!retval) {
```

Comment:
Please follow Chocolate Doom's style guide. All the code in this PR is using end-of-line brace style which isn't the style we use. The brace goes on the following line; please fix everywhere.

If you haven't read it already I suggest you take a look at `HACKING.md` which documents the style guide among other things.